packages/lit-dev-content/samples/tutorials/intro-to-lit/11ty-data.cjs.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/**
+ * This file MUST be CJS to be consumed by 11ty's data system.
+ *
+ * These files must be in the root of the tutorial directory and called
+ * 11ty-data.cjs.(js|ts).
+ *
+ * They must also export a getData async function and getManifest async fn.
+ *
+ * Upon creation of one of these files:
+ *
+ * 1. You must create a set of matching tutorial descriptions at
+ *    `/site/tutorials/content/[tutorial-directory-name]/[slug].md`
+ *
+ * 2. Do a call to `await loadTutorialData('[tutorial-directory-name]')` in
+ *    the order that you want it to appear in
+ *    `/site/tutorials/tutorials.11tydata.js`
+ */
+import {TutorialData, TutorialManifest} from '../utils.cjs.js';
+import path from 'path';
+
+/**
+ * Generates a tutorial data object for this tutorial's catalog card. For more
+ * options, see the TutorialData type in /samples/tutorials/utils.cjs.ts
```

Comment:
Nit: Should be able to directly link with:

```suggestion
 * options, see {@linkcode TutorialData}.
```

Reply:
TIL